spec/models/user_spec.rb

```diff
@@ -1,25 +1,24 @@
-require 'test_helper'
-
-class UserTest < ActiveSupport::TestCase
-  test "is valid" do
+require 'rails_helper' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.